README.md

```diff
@@ -0,0 +1,10 @@
+This project is team project for Team “Not A Team” of 2021 Fall CSC207. It is a simple farm game.
+
+The .idea folder and gradle files are auto generated by Android Studio, it might look messy but if
+they were mis-placed, our project could run into error.
```

Comment:
Rephrase:  The .idea folder and Gradle files are auto-generated by Android Studio, it might look messy but if
they were refactored, our project could run into errors.